app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -328,10 +330,17 @@ class NewMessageViewModel @Inject constructor(
         )
     }
 
-    private fun Draft.initLiveData() {
+    private fun Draft.initLiveData(signatures: List<Signature>) {
 
         savedStateHandle[NewMessageActivityArgs::draftLocalUuid.name] = localUuid
 
+        fromLiveData.postValue(
+            UiFrom(
+                signature = signatures.single { it.id == identityId?.toInt() },
+                shouldUpdateBodySignature = false,
```

Comment:
I don't really get why we shouldn't update the body during init.
Is it because the webview isn't loaded or something like this ? 
What happened if we update here ?